### PR TITLE
Update references to Docker across Dev Guide

### PR DESCRIPTION
**Pull request by ahardin-rh**
Per team discussion

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@adellape @bfallonf PTAL

File: dev_guide/builds.adoc [hunk 1]
```diff
@@ -118,9 +118,9 @@ or `*Binary*`, to accept binary payloads. It is possible to have multiple
 sources at once, refer to the documentation for each source type for details.
 <5> The `*strategy*` section describes the build strategy used to execute the
 build. You can specify `*Source*`, `*Docker*` and `*Custom*` strategies here.
-This above example uses the `*ruby-20-centos7*` Docker image that
+This above example uses the `*ruby-20-centos7*` conatiner image that
```
Comment (bfallonf) [inline dev_guide/builds.adoc]: s/conatiner/container

File: dev_guide/builds.adoc [hunk 2]
```diff
@@ -118,9 +118,9 @@ or `*Binary*`, to accept binary payloads. It is possible to have multiple
...
 Source-To-Image will use for the application build.
-<6> After the Docker image is successfully built, it will be pushed into the
+<6> After the conatiner image is successfully built, it will be pushed into the
```
Comment (bfallonf) [inline dev_guide/builds.adoc]: s/conatiner/container

File: dev_guide/builds.adoc [hunk 3]
```diff
@@ -1166,7 +1166,7 @@ executing the build.
 
 Additional files can be provided to the build process via images. Input images
 are referenced in the same way the `*From*` and `*To*` image targets are
-defined. This means both Docker images and image stream tags can be referenced.
+defined. This means both conatiner images and image stream tags can be referenced.
```
Comment (bfallonf) [inline dev_guide/builds.adoc]: s/conatiner/container

File: dev_guide/migrating_applications/S2I_tool.adoc [hunk 4]
```diff
@@ -14,8 +14,8 @@ toc::[]
 == Overview
 
 xref:../../using_images/s2i_images/index.adoc#using-images-s2i-images-index[The
-Source-to-Image (S2I) tool] injects application source code into a Docker image
-and the final product is a new and ready-to-run Docker image that incorporates
+Source-to-Image (S2I) tool] injects application source code into a conatiner image
```
Comment (bfallonf) [inline dev_guide/migrating_applications/S2I_tool.adoc]: s/conatiner/container

File: dev_guide/migrating_applications/support_guide.adoc [hunk 5]
```diff
@@ -137,7 +137,7 @@ Databases] section of the Database Applications topic.
 |===
 |v2 |v3
 
-|force_clean_build  |There is a similar concept in v3, as *noCache* field in *buildconfig* forces the Docker build to rerun each layer. In the S2I build, the *incremental* flag is false by default, which indicates a *clean build*.
+|force_clean_build  |There is a similar concept in v3, as *noCache* field in *buildconfig* forces the conatiner build to rerun each layer. In the S2I build, the *incremental* flag is false by default, which indicates a *clean build*.
```
Comment (bfallonf) [inline dev_guide/migrating_applications/support_guide.adoc]: s/conatiner/container

**Comment by bfallonf** [comment]
@ahardin-rh a few s/conatiner/container 's, but fine otherwise!

**Comment by ahardin-rh** [comment]
@bfallonf Whoops! These are fixed.

**Comment by ahardin-rh** [comment]
No rev history needed

**Pull request merged**